src/app/containers/Image/index.jsx

```diff
@@ -20,15 +20,17 @@ const getCopyright = copyrightHolder => {
   return copyrightHolder;
 };
 
-const getIChefURL = (originCode, locator) => {
+const getIChefURL = (originCode, locator, width) => {
```

Comment:
Ermm I dont think we should do this. This means that the ichef width is the size of the image width right? This means that unless your image matches `?:1024|1536|2048|\d{2,3}` it will fail. 

Also, going live, i would much rather we dont risk sending `2048` wide images to a mobile device.

Reply:
This was done before I stepped into the PR, but I'm guessing it's a useful means of testing different image sizes now that we want to check that images don't scale beyond their additional width.

Agreed we shouldn't allow any sizes that ImageChef doesn't support. What would you suggest?

Reply:
For now, leave ichef as it is untill https://github.com/bbc/simorgh/issues/971 is done